### PR TITLE
[DOC] Visualization type - Region Map Move Page

### LINKED ISSUE #2317: [DOC] Visualization types - Using Region Map
**What do you want to do?**
 
- [X] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 Update front matter to move page; related to TOC restructure

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
#992

### PR DESCRIPTION
**Pull request by vagimeli**
Signed-off-by: vagimeli <vagimeli@amazon.com>

### Description
Move page to new site page; part of TOC restructure; update front matter

### Issues Resolved
#2317 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@Naarcha-AWS Will you review the front matter structure for accuracy? This PR is related to the TOC restructure. Minor edits made to content.

File: _dashboards/visualize/geojson-regionmaps.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: Region map visualizations
-nav_order: 7
+title: Using Region Map
+parent: Visualization types
```
Comment (Naarcha-AWS) [inline _dashboards/visualize/geojson-regionmaps.md:4]: Do we have page called Visualization Types created yet?

Comment (vagimeli) [reply _dashboards/visualize/geojson-regionmaps.md:4]: @Naarcha-AWS Yes, see PR #2216

File: _dashboards/visualize/geojson-regionmaps.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+grand_parent: Visualize
+nav_order: 70
```
Comment (Naarcha-AWS) [inline _dashboards/visualize/geojson-regionmaps.md:6]: Add the following underneath:

```
redirect_from:
  - /dashboards/geojson-regionmaps/
```

**Review by Naarcha-AWS — APPROVED** [review]

**Comment by vagimeli** [comment]
Closing this issue to consolidate several move pages and link update issues into one issue.

**Pull request closed without merging**